Source/BugsnagCrashReport.h

```diff
@@ -37,9 +37,9 @@ NSString *_Nonnull BSGFormatSeverity(BSGSeverity severity);
 @interface BugsnagCrashReport : NSObject
 
 /**
- *  Create a new crash report from a JSON crash report generated by KSCrash
+ *  Create a new crash report from a JSON crash report generated by BugsnagCrashSentry
```

Comment:
I don't think renaming this makes sense yet, since the underlying interface hasn't changed.